report-viewer/src/model/factories/OverviewFactory.ts

```diff
@@ -68,6 +81,14 @@ export class OverviewFactory {
     );
   }
 
+  static compareVersions(jsonVersion: Version, reportViewerVersion: Version) {
+    if(jsonVersion.major !== reportViewerVersion.major ||
+       jsonVersion.minor !== reportViewerVersion.minor ||
+       jsonVersion.patch !== reportViewerVersion.patch) {
+        console.warn("The result's version tag does not fit the report viewer's version. Trying to read it anyhow but be careful.")
+       }
+  }
+
```

Comment:
If we simplify it to String, we can use `semver` to compare the version here

https://stackoverflow.com/questions/6832596/how-can-i-compare-software-version-number-using-javascript-only-numbers

Reply:
I don't think that replacing a quite simple solution with yet another dependency is a that good approach :D